lib/index.js

```diff
@@ -114,27 +115,21 @@ Nanigans.prototype.track = function(track) {
       params.products.value = data.value;
       params.products.sku = data.sku;
       params.products = qs.stringify(params.products);
-      this.load('purchase', params);
+      params.userId ? this.load('purchase', params) : this.load('purchase_no_user_id', params);
       break;
     case 'user':
       switch (event.name) {
-      case 'product':
```

Comment:
It was not used (ie it wasn't possible to select this event type in the Segment UI as seen [here](https://cl.ly/0x2N390l2O1i) in metadata as seen [here](https://cl.ly/1r18023N0t0o)). Additionally, the Tong from Nanigans said it's not required as [seen here](https://cl.ly/0z0w1G3Q2m33).